schema.graphql

```diff
@@ -38,6 +38,35 @@ type Query {
         "Maximum number of recommendations to return in {Slate.recommendations}, defaults to 10"
         recommendationCount: Int = 10
     ): SlateLineup
+
+    """
+    Request a specific `RankedCorpusItems` by ID: ultimately replaces `getSlate`
+    """
+    getRankedCorpusItems(
+        id: String!
+    ): RankedCorpusItems
+}
+
+type RankedCorpusItems @key(fields: "id") {
+    id: ID!
+    description: String!
+    displayName: String!
+    experimentId: String!
+    corpusItems: [CorpusItem!]
```

Comment:
Important: Every corpus item will require its own `recommendation_id` or other ID to uniquely track the engagement of this item. We can make this follow-up work when we start delivering recommendation results to the data warehouse.